app/helpers/shared_helper/ab_list_helper.rb

```diff
@@ -0,0 +1,141 @@
+module SharedHelper::AbListHelper
+  private
+
+  def shared_button_summary(custom_button, sb_items, resolve)
+    summary = [
+      shared_button_basic_info(custom_button, sb_items),
+      object_details(custom_button, resolve),
+      object_attribute(custom_button, resolve),
+      visibility_expression(custom_button),
+      attribute_value_pair(resolve),
+      visibility(custom_button)
+    ]
+    safe_join(summary)
+  end
+
+  def shared_button_basic_info(custom_button, sb_items)
+    style = custom_button.options[:button_color]&.to_s
+
+    display_for = custom_button.options[:display_for]
+    display_how_map = {'single' => _('Single entity'), 'list' => _('List'), 'both' => _('Single and list')}
+    display_for_value = display_how_map[display_for] if display_for.present?
+
+    submit_how = custom_button.options[:submit_how]
+    submit_how_map = {'all' => _('Submit all'), 'one' => _('One by one')}
+    submit_for_value = submit_how_map[submit_how] if submit_how.present?
+
+    rows = [
+      row_data(_('Button Type'), custom_button.options[:button_type].capitalize),
+      row_data(_('Name'), custom_button.name),
+      row_data(_('Display on button'), {:input => "checkbox", :name => "display", :checked => true, :disabled => true}),
+      row_data(_('Description'), custom_button.description),
+      row_data(_('Image'), custom_button.options[:button_icon], style, :icon => true),
+      row_data(_('Dialog'), sb_items[:dialog_label]),
+      row_data(_('Open URL'), custom_button.options[:open_url]),
+      row_data(_('Display for'), display_for_value&.capitalize),
+      row_data(_('Submit'), submit_for_value&.capitalize),
+    ]
+
+    miq_structured_list({
+                          :title => _('Basic Information'),
+                          :mode  => _('run_enablement_summary'),
```

Comment:
`enablement` could be a name provided by a user.
We need to change this to a more generic name. maybe `button` will be ok for now.